loader/src/sources/prepmod/index.js

```diff
@@ -0,0 +1,427 @@
+/**
+ * Load data from PrepMod's SMART Scheduling Links API
+ * https://github.com/smart-on-fhir/smart-scheduling-links/
+ *
+ * PrepMod maintains separate deployments for each customer, so we have to
+ * contact at least one (and sometimes several) hosts in each state.
+ */
+
+const got = require("got");
+const Sentry = require("@sentry/node");
+const { Available, LocationType } = require("../../model");
+const prepmodHostsByState = require("./hosts");
+const { HTTPError } = require("got");
+
+const API_PATH = "/api/smart-scheduling-links/$bulk-publish";
+const MANIFEST_TIMEOUT = 5 * 60 * 1000;
+const HL7_SERVICE_TYPE_SYSTEM =
+  "http://terminology.hl7.org/CodeSystem/service-type";
+const VTRCKS_SYSTEM = "https://cdc.gov/vaccines/programs/vtrcks";
+const CAPACITY_EXTENSION =
+  "http://fhir-registry.smarthealthit.org/StructureDefinition/slot-capacity";
+const BOOKING_DEEP_LINK_EXTENSION =
+  "http://fhir-registry.smarthealthit.org/StructureDefinition/booking-deep-link";
+const PRODUCT_EXTENSION =
+  "http://fhir-registry.smarthealthit.org/StructureDefinition/vaccine-product";
+const DOSE_EXTENSION =
+  "http://fhir-registry.smarthealthit.org/StructureDefinition/vaccine-dose";
+
+const PRODUCTS_BY_CVX_CODE = {
+  210: "astra_zeneca",
+  207: "moderna",
+  211: "novavax",
+  208: "pfizer",
+  212: "jj",
+};
+
+// Use symbols to link slots -> schedules -> locations to avoid circular
+// references when serializing or logging data.
+const locationReference = Symbol("schedule");
```

Comment:
Doesn't affect functionality, but would increase clarity:

```suggestion
const locationReference = Symbol("location");
```

Reply:
Gah! This must have been a copy/paste error.